### ISSUE TITLE
google drive v3.d.ts is broken

### ISSUE THREAD

**Issue by ZingerDerGOD** [google/google-api-nodejs-client#1178]
Hi dear Google!

Your **google drive node client** codez do not work...

Your test do not work, and if they do - they suck.

Just to get things clear - this "problem" affects only TS projects.
Happy weakly-typed JS people just missed that pleasure.

Boy I hope you can follow this from these clues:
```
get(params?: Params$Resource$Files$Get, options?: MethodOptions): AxiosPromise<any>;//Schema$File>;
```

and 

```
    interface Params$Resource$Files$Get {
        /**
         * Auth client or API Key for the request
         */
        auth?: string | OAuth2Client | JWT | Compute | UserRefreshClient;
        /**
         * Whether the user is acknowledging the risk of downloading known malware
         * or other abusive files. This is only applicable when alt=media.
         */
        acknowledgeAbuse?: boolean;
        /**
         * The ID of the file.
         */
        fileId?: string;
        /**
         * Whether the requesting application supports Team Drives.
         */
        supportsTeamDrives?: boolean;

        //Huh? Missing? Still works though...
        alt?: string;
    }
```
That's what I had to do to get at very least an example code running...

Some technical details:
File:
```
./apis/drive/v3.d.ts
```
Package version: 
```
"googleapis": "^30.0.0"
```

Boy I hope you'll get this super riddle resolved some day, so that we could use your modules in CI.

Have a nice day, thanks in advance!

**Comment by ZingerDerGOD** [comment]
Just to be completely clear - that's the example code adaptation that does not compile basically:
```
            const drive = google.drive({version: 'v3', auth});
            let dest = {};
            let chunks = [];
            const res = await drive.files.get(
                {
                    fileId: fileDescriptor.id, 
                    alt: 'media'
                },
                {
                    responseType: 'stream'
                }
            );
            res.data
            .on('data', d => {
                chunks.push(d);
            })
            .on('end', () => {
                resolve(Buffer.concat(chunks));
            })
            .on('error', err => {
                reject(err);
            });
```

**Comment by JustinBeckwith** [comment]
Greetings!  Fairly certain we fixed this in 31.0.2.  Please take a look and let me know if you're running into any issues!
